### DISCUSSION TITLE
Handling data from test/dev epochs

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_experiments#197] [General]
Currently on ceph, we do not distinguish between "real" experiment data, and "test/dev" epoch data.

@glopesdev had a proposal to add a directory (something called `quarantine_data/` or `dev_data/` or `test_data/`) directory to ceph that is parallel to `data/` that would be written to during test/dev modes.

This was brought up in a discussion that centered on being able to run epochs while the repo is dirty (which is currently not allowed).

**Comment by glopesdev** [comment]
Related to https://github.com/SainsburyWellcomeCentre/aeon_experiments/issues/186

**Comment by jkbhagatio** [comment]
I think we've decided not to do this, and just take an approach where the folder names are the branch names which are the exp names?

**Comment by jkbhagatio** [comment]
> I think we've decided not to do this, and just take an approach where the folder names are the branch names which are the exp names?

Yes, so closing.
